bin/node/rpc/Cargo.toml

```diff
@@ -31,3 +32,4 @@ sp-blockchain = { version = "2.0.0-rc3", path = "../../../primitives/blockchain"
 sc-finality-grandpa = { version = "0.8.0-rc3", path = "../../../client/finality-grandpa" }
 sc-finality-grandpa-rpc = { version = "0.8.0-rc3", path = "../../../client/finality-grandpa/rpc" }
 sc-rpc-api = { version = "0.8.0-rc3", path = "../../../client/rpc-api" }
+sc-rpc = { version = "2.0.0-rc3", path = "../../../client/rpc" }
```

Comment:
Is this needed?

Reply:
Currently it's used due to using a concrete `sc_rpc::Metadata` instead of the generic one used before. I'll try again to see if we can switch back to a generic type